src/services/CertificateGenerationService.ts

```diff
@@ -87,7 +87,7 @@ class CertificateGenerationService {
                     dateOfIssue: moment().format("D MMMM YYYY"),
                     certificateType: certificateTypes[vehicleTestRes].split(".")[0],
                     fileFormat: "pdf",
-                    fileName: `${testResult.testResultId}_${testResult.vin}_${testResult.order.current}.pdf`,
+                    fileName: `${testResult.testTypes.testNumber}_${testResult.vin}_${testResult.order.current}.pdf`,
```

Comment:
I think the story is not mentioning the order.current, is that something we want to keep?
It is mentioning testNumber_VIN.pdf format